src/Uno.UI.Wasm/ts/MonoSupport.ts

```diff
@@ -30,6 +30,7 @@ namespace MonoSupport {
 				if (!jsCallDispatcher._isUnoRegistered) {
 					jsCallDispatcher.registerScope("UnoStatic", Uno.UI.WindowManager);
 					jsCallDispatcher.registerScope("UnoStatic_Windows_Storage_StorageFolder", Windows.Storage.StorageFolder);
+					jsCallDispatcher.registerScope("UnoStatic_Windows_Storage_ApplicationDataContainer", Windows.Storage.ApplicationDataContainer);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5656018)